src/features/bumpkins/components/SkillBadges.tsx

```diff
@@ -32,9 +45,38 @@ export const SkillBadges: React.FC<{ inventory: Inventory }> = ({
     return null;
   }).filter(Boolean);
 
+  const skills = SKILLS.map((skill) => {
+    if (bumpkin.skills[skill]) {
+      return (
+        <img
+          key={skill}
+          src={BUMPKIN_SKILL_TREE[skill].image}
+          alt={skill}
+          style={{
+            opacity: 0,
+            marginRight: `${PIXEL_SCALE * 2}px`,
+            marginBottom: `${PIXEL_SCALE * 2}px`,
+          }}
+          onLoad={(e) => setImageWidth(e.currentTarget)}
+        />
+      );
+    }
+
+    return null;
+  }).filter(Boolean);
+
+  if (skills.length === 0) {
```

Comment:
Having these checks seperate isn't the best way to do this because if a user has `badges` but no `skills` we will return early and not see the `badges`. 

How about something like this?

```suggestion
const totalSkills = [...badges, ...skills];

if (totalSkills.length === 0) return null;
  
return (
    <div className="flex flex-wrap items-center">
      {totalSkills}
    </div>
);
```